fleet_adapter_ecobot/fleet_adapter_ecobot.py

```diff
@@ -190,6 +190,9 @@ def _add_fleet_robots():
                     starts = []
                     time_now = adapter.now()
 
+                    x,y = transforms['ecobot_to_rmf'].transform([ecobot_pos[0],ecobot_pos[1]])
+                    position = [x, y, 0]
```

Comment:
You should also transform the yaw. See https://github.com/open-rmf/fleet_adapter_ecobot/blob/be5d1692d031fa2eb597e4002898b978fc6c3025/fleet_adapter_ecobot/EcobotCommandHandle.py#L412-L428

Otherwise you're forcing the fleet adapter to rotate each robot to 0deg when the responsive wait begins.

Reply:
fixed https://github.com/open-rmf/fleet_adapter_ecobot/pull/3/commits/6b414dc2df59a8838fc08369ca9b86f70d804b96